light_curves/light_curve_generator.md

```diff
@@ -131,9 +135,9 @@ get_yang_sample(coords, labels)   #2018ApJ...862..109Y
 # a balance between speed of running the light curves and whatever
 # the ML algorithms would like to have
 
-# num_normal_QSO = 5000
-# zmin, zmax = 0, 10
-# randomize_z = False
+#num_normal_QSO = 30
+#zmin, zmax = 0, 10
+#randomize_z = False
```

Comment:
are these changes intended?

Reply:
From my perspective, it doesn't matter what the values of these are since they are commented out.  I do change them when I am working on the code and testing different sample sizes, and then most often forget what they originally were and leave them at whatever the last value I tried.  Does that cause a problem?  Or is it the spaces between the # and the commands that is concerning?

Reply:
It's the adding unrelated and unnecessary changes. I see why it happens, but it would be nice to figure out a convenient way to change the workflow so only intended and necessary changes get added. 
I added this topic to my list for the ipac visit.

Reply:
yes!  I don't know how to do this, so perfect to discuss later this month.

Reply:
I've seen a new (to me anyway) "git" tab on the LHS of the Fornax console that gives GUI access to changed and staged files, etc. Maybe there's a way to select and commit individual lines/changes from there? I haven't played around with it at all but seems worth checking out.

Reply:
Yeap, I put on the list for my visit to experiment something out.